Makefile

```diff
@@ -454,6 +454,13 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specifi checks
```

Comment:
What do you think about including `vet` into `check-style`? This way devs only need to run one command and the circleci config is simpler.

Reply:
good point.